lib/Item.php

```diff
@@ -222,6 +233,13 @@ private function deleteFile(): void {
 		}
 	}
 
+	/**
+	 *    * Mark infected file
+	 */
+	private function markFile(): void {
+		$this->file->move($this->file->getPath() . " [\u{2623} VIRUS DETECTED \u{2620}]");
```

Comment:
```suggestion
		$this->file->move($this->file->getPath() . " [VIRUS DETECTED]");
```

IMO, the unicode chars are a bit too much.

Reply:
What if the files already marked? Will it get the same suffix again on the second scan?

E.g. `some_file [VIRUS DETECTED] [VIRUS DETECTED]`